src/hu_stuff.c

```diff
@@ -1645,8 +1645,7 @@ void HU_Ticker(void)
   {
     if (hud_type == HUD_TYPE_CRISPY)
     {
-      if (hud_active > 0)
-        draw_crispy_hud = true;
+      draw_crispy_hud = true;
```

Comment:
This `hud_active` check has always had strange behavior with Crispy HUD. Basically, you have to hit `F5` twice to hide the HUD, then once more to show it, repeat. With the change I made here, `F5` brings up the Crispy HUD if it's enabled but doesn't hide it again. Do we want to toggle Crispy HUD with `F5` still? If so, some other logic will be needed here.

Reply:
It seems like the `F5` behavior could use a thorough review, possibly in a separate PR. It seems natural to want it to toggle between status bar, and one or both huds, or even the modes possibly.

Reply:
> hit `F5` twice to hide the HUD, then once more to show it, repeat

Well, it switched between off, compact and distributed mode, the latter two were plain identical if widescreen rendering was not enabled.

Reply:
Okay, let me restore that behavior at the very least. I don't want to change that functionality.

Reply:
I handle this idea properly in a different PR.

Reply:
> I'll handle this idea properly in a different PR.

Still makes perfect sense as it is now, because "Widescreen" affects Boom widgets and the Crispy HUD's widescreen alignment is indeed its "distributed mode".